### PR TITLE
Add iscsi information parser

### PR DESCRIPTION
**Pull request by hassanshabbirahmed**
Adding a very simple iscsi status parser

### CODE REVIEW

**Comment by SuperQ** [comment]
If you look at the way other sysfs collectors work, you can see that they implement the `filepath.Glob()` in the `fs.go` file.  Please take a look at how `net_class.go` is implemented. This is the required design pattern.

**Comment by hassanshabbirahmed** [comment]
Hey @SuperQ I've morphed it according to the design pattern you mentioned. Would you mind taking a look again.

**Comment by SuperQ** [comment]
It looks like your fork is far enough behind the current master that you're missing the change from Travis to CircleCI.

Would you please sync up your fork and rebase this change?

**Comment by SuperQ** [comment]
Also, this change is missing tests and fixtures like net_class_test.go has.

**Comment by pgier** [comment]
I think it might make sense to move this code into the new iscsi package added in #171 
Also, this needs some unit tests.

File: sysfs/iscsi.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
+// Copyright 2015 The Prometheus Authors
```
Comment (pgier) [inline sysfs/iscsi.go]: Should be 2019

**Comment by AvengerMoJo** [comment]
@hassanshabbirahmed this is reading from the iscsi initiator point of view. My node_exporter module is from the iscsi target point of view. I think we can merge the two into one.

**Comment by discordianfish** [comment]
I'm going to close this PR. Let's reopen if somebody has time to move it to the iscsi package.

**Pull request closed without merging**